sdk/tests/types/output/account.rs

```diff
@@ -63,7 +63,7 @@ fn builder() {
 
     assert_eq!(
         output.amount(),
-        Output::Account(output.clone()).rent_cost(protocol_parameters.rent_structure())
+        Output::Account(output.clone()).minimum_amount(protocol_parameters.storage_score_parameters())
```

Comment:
I know it's only a test but we don't need to wrap/clone, right? Trying to prevent this pattern for proliferating if it's not necessary.